cmd/kata-manager/kata-manager.sh

```diff
@@ -316,6 +327,11 @@ exec_document()
 	# create the script
 	"${doc2script}" "${file}" "${install_script}" "${msg}"
 
+	# set the kata release in the script
+	if [ ! -z "$release" ]; then
```

Comment:
`-n` is equals to `! -z`

Reply:
thanks @devimc, this part ended up being removed